<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LedCubeDataSourceClass</class>
 <widget class="QMainWindow" name="LedCubeDataSourceClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>483</width>
    <height>406</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>LedCubeDataSource</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0" colspan="2">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QPushButton" name="bSendMagic">
        <property name="text">
         <string>send magic</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="bSendFrame">
        <property name="text">
         <string>send frame</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="bSendOther">
        <property name="text">
         <string>send other</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label">
      <property name="text">
       <string>Sent:</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>Receoved:</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QPlainTextEdit" name="eSent">
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QPlainTextEdit" name="eReceived">
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>483</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="ledcubedatasource.qrc"/>
 </resources>
 <connections/>
</ui>
